ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/private/shared-vpc-phz/sts/deprovision/cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts-deprovision-chain.yaml

```diff
@@ -0,0 +1,11 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-private-shared-vpc-phz-sts-deprovision
+  steps:
+  - chain: cucushift-installer-rehearse-aws-ipi-private-deprovision
+  - ref: aws-deprovision-route53-private-hosted-zone
```

Comment:
Here we are not using `cucushift-installer-rehearse-aws-ipi-byo-route53-deprovision` chain?

Reply:
> Here we are not using `cucushift-installer-rehearse-aws-ipi-byo-route53-deprovision` chain?

Right, `gather-installer-boostrap-logs-in-bastion` and `aws-deprovision-s3buckets` are required by private cluster.

Reply:
ah, got it.